crates/manifest/tests/ui/maximal.json

```diff
@@ -59,8 +59,8 @@
       "exclude_files": [
         "**/secret"
       ],
-      "allowed_http_hosts": [
-        "example.com"
+      "allowed_outbound_hosts": [
+        "https://example.com:443"
```

Comment:
Requiring _both_ a port and a scheme does seem pretty odd, especially for http(s).